torchgeo/samplers/batch.py

```diff
@@ -95,16 +95,9 @@ def __iter__(self) -> Iterator[List[BoundingBox]]:
             # Choose random indices within that tile
```

Comment:
In the `__init__` method, I think we can skip calling `_to_tuple` now that we call it in the `get_random_bounding_box` method instead.

Reply:
Sure